Decred Wallet/Features/Wallet Utils/Syncer.swift

```diff
@@ -208,6 +229,29 @@ class Syncer: NSObject, AppLifeCycleDelegate {
             self.networkLastActive = nil // Network is active at this point.
         }
     }
+
+    @objc private func reinstateBackgroundTask() {
+        if backgroundTask ==  .invalid {
+            registerBackgroundTask()
+        }
+    }
+
+    private func registerBackgroundTask() {
+        backgroundTask = UIApplication.shared.beginBackgroundTask { [weak self] in
+            self?.endBackgroundTask()
```

Comment:
Add a print statement here: `background task expired`

Reply:
This is already handled in the `endBackgroundTask()` method where there is a print statement indicated the time it expired.